data/json/items/comestibles.json

```diff
@@ -1129,8 +1129,8 @@
         "volume" : 1,
         "phase" : "liquid",
         "charges" : 7,
-        "flags": [ "EATEN_COLD" ],
-        "fun" : 12
+        "fun" : 12,
+        "flags": [ "EATEN_COLD" ]
```

Comment:
I think I added that because all wine is served cold (as far as I know), and the description says it's a higher proof wine. But after a little more research, it looks like I was wrong. From [Wikipedia](https://en.wikipedia.org/wiki/Brandy):
`Brandy is traditionally served at room temperature....`

So I'll remove the flag from brandy.